.github/actions/build-platform-docker/action.yml

```diff
@@ -84,7 +87,7 @@ runs:
       with:
         context: .
         file: ${{ inputs.dockerfile }}
-        tags: ${{ steps.get-image.outputs.result }}
+        tags: ${{ steps.get-image.outputs.tags }}
```

Comment:
❌ I'm not sure whether this is exact the same or comparable. 

Before it would have been `camunda/zeebe:8.5.0` here it would be just `8.5.0` https://github.com/docker/metadata-action?tab=readme-ov-file#basic

Reply:
@Zelldon 
the docker/metadata-action outputs tags under form `{image}:{tag}` 
Example in [zeebe-ci,](https://github.com/camunda/camunda/actions/runs/9580383802/job/26415005356?pr=19521) you can see the tags that we pass to `build-push-action`
```
Run docker/build-push-action@v6
  with:
    context: .
    file: Dockerfile
    tags: localhost:5000/camunda/zeebe:current-test
    load: false
    push: true
    platforms: linux/amd64
    provenance: false
    build-args: DISTBALL=dist/target/camunda-zeebe-8.6.0-SNAPSHOT.tar.gz
```

Reply:
ok then I was confused by their docs
![2024-06-19_13-51](https://github.com/camunda/camunda/assets/2758593/791e5182-3756-418a-931b-5641eb03b69a)